data/multiregion.json

```diff
@@ -7,9 +7,9 @@
     "is_dirty": false
   },
   "model_git_info": {
-    "sha": "3c234319406d624ec66830fe4a656bc5253cb49d",
-    "branch": "main",
```

Comment:
Do you mind reverting the `data/` changes and then just kicking off a new dataset update from main once this is merged?

Probably doesn't really matter but as a matter of principle it's nice to always be building our data / snapshots from `main` bits.

Reply:
yep! will do, also I don't think we would want this data as it includes the backlogged data